src/libs/GlobalProvider.tsx

```diff
@@ -0,0 +1,25 @@
+import { createWS } from "@solid-primitives/websocket";
+import { createSignal } from "solid-js";
+
+import { GlobalContext } from "~/libs/context";
+import { setStore, store } from "~/libs/store";
+
+import type { MessageType } from "~/types";
+
+export default function GlobalProvider(props: any) {
```

Comment:
This shouldn't be `any`, but for now just add a `TODO` comment then we can come back to this.